xapian-applications/omega/templates/query

```diff
@@ -88,7 +88,7 @@ $list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;
 <br><small>Search took $time seconds</small>
 <table>
 $hitlist{<tr><td valign=top>
-$set{clickurl,$if{$opt{clicklog},$or{$env{SCRIPT_NAME},omega.cgi}?FMT=clicklog&DB=$url{$dbname}&QUERYID=$opt{queryid}&DOCID=$url{$id}&URL=$url{$field{url}},$field{url}}}
+$set{clickurl,$if{$opt{clicklog},$or{$env{SCRIPT_NAME},omega.cgi}?FMT=clicklog&DB=$url{$dbname}&QUERYID=$opt{queryid}&DOCID=$url{$id}&URL=$url{$field{url}},$html{$field{url}}}}
```

Comment:
Ah, I see. I didn't encounter anything unusual with non-clicklog case when I tested these changes on the browser because no URL in my case has `&` in it. Thanks for the explanation; will fix this.